README.md

```diff
@@ -1,11 +1,14 @@
-# README
+# Red Chainer : A deep learning framework
 
-## Name
-
-Red Chainer
+A flexible framework for neural network for Ruby
 
 ## Description
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/chainer`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+It ported python's Chainer with Ruby.
```

Comment:
When the user wants to know something about chainer, how about linking so that it can be found easily.

Reply:
Thank you for your comment.
I added the link of Chainer.